tests/resources/conformance_suites/nl_nt16/fg_nl/11-testcase.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Doesn't look like these testcases were added to the corresponding `index.xml` files.